proto/DEVELOPMENT.md

```diff
@@ -0,0 +1,4 @@
+# Update Protobuf definitions
+
+- Run `./hack/generate_proto` to regenerate protocol buffer code.
```

Comment:
Please remove that and just use the maven plugin

Reply:
http://os72.github.io/protoc-jar-maven-plugin/

Reply:
We decided to use the compiler directly because we need `protoc` anyway for the `control-plane`. 
The compiler must generate files into the directory `src/main/generated` of the `core` module using the package `dev.knative.eventing.kafka.broker.core.config`.

Reply:
SGTM, what matters is that we keep separated the generated code from the src code :wink: